tests/retrieve/sentence_vectorizer/test_openai_vectorizer.py

```diff
@@ -0,0 +1,24 @@
+import os
+import pytest
+
+from dspy import Example
+from dsp import OpenAIVectorizer
+
+
+@pytest.mark.skipif(os.getenv('OPENAI_API_KEY') is None, reason="Skipping this test because OPENAI_API_KEY is not set")
+def test__call__():
+    vectorizer = OpenAIVectorizer()
```

Comment:
I would recommend mocking the object over here instead of having the test skipping, it would maintain it for later usage and not base on the OPENAI_API_KEY.